### PR TITLE
Adrres Xiaomi global region rumors

### PR DESCRIPTION
**Pull request by lpapadakos**

### CODE REVIEW

File: brands/xiaomi/README.md [hunk 1]
```diff
@@ -14,23 +14,13 @@ Also, check out [Xiaomi-HyperOS-BootLoader-Bypass] by [MlgmXyysd](https://github
...
-- You have to use HyperOS and report at least 1 bug per day. (????)
-- You have to submit at least 1 HyperOS suggestion per month. (????)
-- You must be an active Xiaomi Community user.
-
-Additional BS applies:
-- The permission to unlock is temporary and is granted for 1-year only.
-- You can only unlock 3 devices per-year.
-- You will not receive OTA updates.
-- your warranty will be void.
+[The bootloader CANNOT be unlocked on devices of the Mainland China region](https://gsmchina.com/xiaomi-will-no-longer-allow-bootloader-unlocking-5618/).
```
Comment (melontini) [inline brands/xiaomi/README.md:17]: I believe that the China thing started ~a month ago, so a more recent article would fit better. This one still talks about the old, short-lived requirements.

Comment (melontini) [inline brands/xiaomi/README.md:17]: This should be a reference link.

e.g. 

```
[The bootloader CANNOT be unlocked on devices of the Mainland China region][bootloader-in-mainland]

...
[Xiaomi-bootloader]:https://github.com/lrh2000/Xiaomi-bootloader
[bootloader-in-mainland]:https://gsmchina.com/xiaomi-will-no-longer-allow-bootloader-unlocking-5618/
```

File: brands/xiaomi/README.md [hunk 2]
```diff
@@ -14,23 +14,13 @@ Also, check out [Xiaomi-HyperOS-BootLoader-Bypass] by [MlgmXyysd](https://github
...
 
 ### Global
 
+> [!WARNING]
+> Since October 2024 [there have been rumors that Xiaomi will kill bootloader unlocking on the Global market](https://x.com/chunvn8888/status/1841901853073953254), after having done so for the China market. If you own a Xiaomi device and your warranty has ended or you don't care for it, you should unlock youe device while you can.
```
Comment (melontini) [inline brands/xiaomi/README.md:22]: This should be a reference link. Also, can be shortened to

```patch
- [there have been rumors that Xiaomi will kill bootloader unlocking on the Global market](https://x.com/chunvn8888/status/1841901853073953254)
+ [there have been rumors][reference-link] that Xiaomi will kill bootloader unlocking on the Global market
```

**Review by melontini — CHANGES_REQUESTED** [review]
Thank you. There's a source issue and some link nitpicks.

**Review by melontini — APPROVED** [review]
LGTM.

**Comment by mirh** [comment]
The surprise is here 
https://xiaomitime.com/xiaomi-global-bootloader-unlock-policy-has-changed-20295/

**Pull request merged**